src/cdk-experimental/listbox/listbox-module.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {NgModule} from '@angular/core';
+import {CdkListbox, CdkOption} from './listbox';
+
+const EXPORTED_DECLARATIONS = [CdkListbox, CdkOption];
+@NgModule({
+    exports: EXPORTED_DECLARATIONS,
```

Comment:
Excuse me but it looks like it's indented by 4, not 2 (here and `listbox.ts`).

Reply:
Fixed, thanks.